st2common/st2common/services/coordination.py

```diff
@@ -180,46 +180,3 @@ def get_coordinator():
         COORDINATOR = coordinator_setup()
 
     return COORDINATOR
-
```

Comment:
I assume this commented out code will be removed?

Reply:
Yes, it will be removed.